firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -373,6 +379,29 @@ private boolean isAuthTokenExpired(PersistedFidEntry persistedFidEntry) {
   private long currentTimeInSecs() {
     return TimeUnit.MILLISECONDS.toSeconds(clock.currentTimeMillis());
   }
+
+  interface CombinedListener extends OnCompleteListener<Void> {}
```

Comment:
don't bother including this, just have AwaitListener extend OnCompleteListener directly

Reply:
Done. Thanks.